docs/introduction_tutorial/intro_tutorial_part1.ipynb

```diff
@@ -4,33 +4,35 @@
    "cell_type": "markdown",
```

Comment:
<p>Note to remove the comment ("to me this doesn't make sense") or else add a clarification here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pangeo-forge/pangeo-forge-recipes/pull/295/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>